x-pack/plugin/security/qa/operator-privileges-tests/src/main/java/org/elasticsearch/xpack/security/operator/actions/RestGetActionsAction.java

```diff
@@ -41,19 +33,7 @@ public String getName() {
     @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
Both `@SuppressForbidden` and `@SuppressWarnings` do not seem needed with the proposed changes.